truss/templates/shared/extra_config_resolver.py

```diff
@@ -0,0 +1,21 @@
+from pathlib import Path
+from typing import Optional
+
+
+class ExtraConfigResolver:
+    EXTRA_CONFIG_MOUNT_DIR = "/etc/extra_config"
+
+    @staticmethod
+    def get_config_value(config_key_name: str) -> Optional[str]:
```

Comment:
Is there any chance we could make this a typed function like
```
T = TypeVar("T", pydantic.BaseModel)

def get_config_value(key: str, model_cls: Type[T]) -> T:
 ...
```

Reply:
Marius and I synced - since we can't guarantee that the file contents of `/etc/dynamic_config/key` will be anything other than a string (or stringified JSON) and since `/etc/dynamic_config/chainlet_url_map` contains a url mapping of chainlet name to predict url that doesn't directly map to a defined type in `truss_chains.definitions`, we'll keep the logic here simple - read from the  `/etc/dynamic_config/key` and return a string of its contents